package.json

```diff
@@ -44,6 +43,10 @@
     "eslint-plugin-prettier": "2.2.0",
     "execa": "0.8.0",
     "jest": "21.0.1",
+    "mocha": "3.5.0",
     "prettier": "1.7.4"
+  },
+  "peerDependencies": {
+    "mocha": "*"
```

Comment:
`*` is never an acceptable value on the RHS of package.json; we should explicitly list each major version we support, ie `^3 || ^4 || ^5 || ^6 || ^7` etc.

Reply:
What about "3+" then? Seems like a pretty legit semver value to me.

Reply:
`>= 3` is valid, but has the same problem that if mocha 8, or 12, or 25 comes out, and doesn't work with us, we'll be falsely claiming it works.

Only explicitly tested mocha versions should be allowed in peer deps (which are a hard install requirement in `npm ls`, npm 7+, `yarn list`, and yarn 2+)